dao/src/test/java/se/fortnox/reactivewizard/db/ParameterizedQueryTest.java

```diff
@@ -60,7 +67,7 @@ public void shouldResolveParametersWithoutAnnotationFromQuery() throws SQLExcept
     @Test
     public void shouldThrowExceptionIfUnnamedParamsUsedInQuery() {
         try {
-            dao.unnamedParameters("myid", "myname").toBlocking().singleOrDefault(null);
+            dao.unnamedParameters("myid", "myname").blockFirst();
```

Comment:
We could use StepVerifier to assert exceptions.

Reply:
The method seems to just throw the exception instead of error signaling. We can use Assertj though.